packages/components/tsconfig.json

```diff
@@ -16,8 +16,9 @@
         "noUnusedLocals": true,
         "noUnusedParameters": true,
         "noFallthroughCasesInSwitch": true,
-        "baseUrl": "."
+        "baseUrl": ".",
+        "types": ["cypress", "@testing-library/cypress"]
     },
-    "include": ["src/**/*"],
+    "include": ["src/**/*", "cypress"],
```

Comment:
I think we need to create a `tsconfig.lib.json` to feed into `dts` to distinguish between the types we need to build and ship, and those who are now overlapping with Cypress.

Thoughts?